x/evm/keeper/keeper.go

```diff
@@ -178,8 +178,8 @@ func (k Keeper) GetChainConfig(ctx sdk.Context) (types.ChainConfig, bool) {
 
 // SetChainConfig sets the mapping from block consensus hash to block height
 func (k Keeper) SetChainConfig(ctx sdk.Context, config types.ChainConfig) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefixChainConfig)
+	store := ctx.KVStore(k.storeKey)
 	bz := k.cdc.MustMarshalBinaryBare(config)
 	// get to an empty key that's already prefixed by KeyPrefixChainConfig
```

Comment:
are we still getting an empty key prefix? i think this might need updating

Reply:
this is in regards to the comment, should update because we are no longer getting an empty key prefix but we are using one that is set in types.KeyPrefixChainConfig